src/handlers.cpp

```diff
@@ -329,19 +329,20 @@ void ImpiTask::send_mar()
 
 void ImpiTask::on_mar_response(Diameter::Message& rsp)
 {
-  Cx::MultimediaAuthAnswer maa(rsp);
+  Cx::MultimediaAuthAnswer *maa = new Cx::MultimediaAuthAnswer(rsp);
```

Comment:
I thought we allowed MAA's to wrap a NULL pointer, but we don't.  I think we can address my leak concerns, though, by always writing the MultimediaAuthAnswer into _maa (disposing of the local maa variable) and then destroying _maa in this object's destructor.  Does that make sense?